cert/aziot-certd/src/lib.rs

```diff
@@ -23,10 +23,13 @@ use aziot_certd_config::{
     EstAuthX509, LocalCa, PreloadedCert,
 };
 
+use async_trait::async_trait;
```

Comment:
As it happens, `rustfmt` recently (?) gained unstable support for enforcing this automatically:

https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports

As with most things rustfmt related, it isn't _exactly_ what we'd like (it doesn't handle the crate.io vs. workspace crate grouping), but if it ever hits stable it might be worth enabling.

Reply:
In-fact, I decided to open an issue on the `rustfmt` repo that references this exact issue:

https://github.com/rust-lang/rustfmt/issues/4693

Reply:
Moved async_trait to the top